### PR TITLE
Educate gas fees

### LINKED ISSUE #2232: Add tooltip to educate to user about gas fee
As a user, I want to understand what is gas fee, why I have to pay it (specially high even for a small transaction) and if it is a fee that MetaMask controls or charges. 

As a user, BEFORE I get crypto in MetaMask, I want to understand that to make a transaction, there's a network fee I have to pay, so that I can make a decision if I will put funds in MetaMask or not.
^ this need is from people who are more familiar with centralized apps that enable moving around funds without network fee. 

Out of scope:
As a user, when I speed up a transaction, I want to clearly see how much extra gas fee I am paying, and the total gas fee for that transaction.

### PR DESCRIPTION
**Pull request by andrepimenta**
**Description**

Adds fiat values and tooltip to educate to user about gas fee.
Check design here: https://www.figma.com/file/J8Gg2fYEY9iB0Mmi0DWk5E/Gas-Education?node-id=0%3A1

**Checklist**

* [x] There is a related GitHub issue

**Issue**

Resolves #2232

### CODE REVIEW

File: app/components/UI/FiatOrders/PaymentMethodSelector/index.android.js [hunk 1]
```diff
@@ -14,19 +14,37 @@ import Title from '../components/Title';
...
+}) {
 	const navigation = useContext(NavigationContext);
 	const transakURL = useTransakFlowURL(selectedAddress);
 
 	const onPressTransak = useCallback(() => {
-		navigation.navigate('TransakFlow', {
-			url: transakURL,
-			title: strings('fiat_on_ramp.transak_webview_title')
-		});
+		const goToTransakFlow = () =>
```
Comment (sethkfman) [inline app/components/UI/FiatOrders/PaymentMethodSelector/index.android.js:29]: Could you pull out the onPressTransak and onPressWyreApplePay to a common area to reduce duplicate code?

**Review by sethkfman — APPROVED** [review]

**Review by ibrahimtaveras00 — CHANGES_REQUESTED** [review]
**Issue 1:**

On a smaller device (iPhone SE) there's no indication that there are multiple pages to scroll through 

image

**Issue 2:**

Warning pops up when changing gas fee

seen here = https://recordit.co/jh6h8ki2rE

Screen Shot 2021-05-28 at 7 11 44 PM

**Issue 3:** 

Not sure if this was changed, but I noticed the URL is different from figma

image

**Issue 4:**

I don't see tooltip nor link format for approve txn type

Screen Shot 2021-05-28 at 8 36 26 PM

If you want an easy way to get approve to pop up tap this deeplink in safari/chrome to open MM app

https://metamask.app.link/approve/0x1d7a9cd8c67e3c74a84bb76cbc1b606e2e460db5@1/approve?address=0x1FDb169Ef12954F20A15852980e1F0C122BfC1D6&uint256=1

**Issue 5:**

The gas fee on Swaps is different than the mock up in figma

image

**Comment by mobularay** [comment]
From today's mid-sprint sync: @jakehaugen and @wachunei will respond to Ibrahim's QA questions, and complete this ticket hopefully while Andre is away.

**Comment by wachunei** [comment]
- **Issue 1**: Added custom styling for small screens 
- **Issue 2**: Fixed 
- **Issue 3**: @jakehaugen can you confirm what is the right url?
- **Issue 4**: Added it!
- **Issue 5**: It seems we are not implementing this because of dotted underline does not exists, @jakehaugen please confirm 🙏

**Comment by jakehaugen** [comment]
Issue 3: The MetaMask link is correct
Issue 5: We decided to go with an underline (not dotted) for the edit link not on swaps, right? If so, let's be consistent and go with the underline. 

For 5, this is what is shown in the mocks: https://www.figma.com/file/J8Gg2fYEY9iB0Mmi0DWk5E/Gas-Education?node-id=72%3A127

**Comment by wachunei** [comment]
**Issue 5**: Fixed

**Review by ibrahimtaveras00 — APPROVED** [review]
All issues have been resolved 👍🏽

**Pull request merged**